### ISSUE TITLE
On the admin API -> While updating the group product the admin panel is showing blank getting exceptions in the console.

### ISSUE THREAD

**Issue by mukulsharma-webkul** [bagisto/rest-api#364]
**Rest API - v2.2.0**

**Bagisto Core - v2.2.0**

**Description -**

On the admin API -> While updating the group product the admin panel is showing blank getting exceptions in the console.

**Please take the reference from the video below -**

https://webkul.chatwhizz.com/share/view-recording/66c5ca40e5f9510e2d206663

**Comment by amit-webkul** [comment]
I have checked and found that when we update grouped product then the API is working fine. I have tested and recorded the video for your reference. [video](https://webkul.chatwhizz.com/share/view-recording/66d1623cf7a4cb09e8af84f9)

**Note:** When we update a grouped product for the first time with an added simple product, the API works fine. However, when we update the same grouped product a second time, we need to include the ID of the added simple product in the requested data, replacing the **_link_0_** key.
